source/frontend/src/models/api/Property.ts

```diff
@@ -43,10 +43,12 @@ export interface Api_Property extends Api_ConcurrentVersion, Api_AuditFields {
 
   isSensitive?: boolean;
   isVisibleToOtherAgencies?: boolean;
+  isProvincialPublicHwy?: boolean;
   isOwned?: boolean;
   isPropertyOfInterest?: boolean;
   pphStatusTypeCode?: string;
   isRwyBeltDomPatent?: boolean;
+  isALR?: boolean;
```

Comment:
I cant find this field on the backend model. Lets keep the Api_* models a 1:1 to the backend